<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleGeneral">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Confirmación Reservas Parque Apícola&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="fondo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>651</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagenes/logo_pca4.png</pixmap>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateReserva">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>8</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="titleDateReserva">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Fecha de pago de la reserva&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="titlenumReserva">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Número de reserva&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="titlenameReserva">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Nombre del titular&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxPlan">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pasadía</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ecocolmena</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Iglú</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hindú</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pirámide</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Castillo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Casa Paisa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Visita</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="titlePlan">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Plan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleID">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Identificación&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idReserva">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameReserva">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="numReserva">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titleCelular">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Celular&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="celularReserva">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titledateLlegada">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Fecha de llegada&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="titledateSalida">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Fecha de salida&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateLlegada">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>3</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>8</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="time">
     <time>
      <hour>3</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateSalida">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>12</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>8</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="time">
     <time>
      <hour>12</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QLabel" name="titlecashReserva">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Valor de reserva&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cashReserva">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titletotalcashReserva">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Valor total del plan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cashtotalReserva">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="boxmetodoPago">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NEQUI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAVIPLATA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Consignación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Efectivo</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="titlemetodoPago">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Método de pago&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titleEmail">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Correo electrónico&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_generar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>fondo</zorder>
   <zorder>titleGeneral</zorder>
   <zorder>dateReserva</zorder>
   <zorder>titleDateReserva</zorder>
   <zorder>titlenumReserva</zorder>
   <zorder>titlenameReserva</zorder>
   <zorder>boxPlan</zorder>
   <zorder>titlePlan</zorder>
   <zorder>titleID</zorder>
   <zorder>idReserva</zorder>
   <zorder>nameReserva</zorder>
   <zorder>numReserva</zorder>
   <zorder>titleCelular</zorder>
   <zorder>celularReserva</zorder>
   <zorder>titledateLlegada</zorder>
   <zorder>titledateSalida</zorder>
   <zorder>dateLlegada</zorder>
   <zorder>dateSalida</zorder>
   <zorder>titlecashReserva</zorder>
   <zorder>cashReserva</zorder>
   <zorder>titletotalcashReserva</zorder>
   <zorder>cashtotalReserva</zorder>
   <zorder>boxmetodoPago</zorder>
   <zorder>titlemetodoPago</zorder>
   <zorder>email</zorder>
   <zorder>titleEmail</zorder>
   <zorder>boton_generar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
